pkg/cel/policies/vpol/engine/provider.go

```diff
@@ -122,7 +123,7 @@ func NewKubeProvider(
 			}
 		},
 	}
-	reconciler := newReconciler(compiler, mgr.GetClient(), polexLister)
+	reconciler := newReconciler(compiler, mgr.GetClient(), polexLister, polexEnabled)
 	err := ctrl.NewControllerManagedBy(mgr).
 		For(&policiesv1alpha1.ValidatingPolicy{}).
 		Watches(&policiesv1alpha1.PolicyException{}, exceptionHandlerFuncs).
```

Comment:
Do we need this it polex is not enabled ?

Reply:
Good catch!